backend/src/main/scala/cromwell/backend/BackendLifecycleActorFactory.scala

```diff
@@ -58,7 +58,11 @@ trait BackendLifecycleActorFactory {
 
   lazy val jobExecutionTokenType: JobExecutionTokenType = {
     val concurrentJobLimit = configurationDescriptor.backendConfig.as[Option[Int]]("concurrent-job-limit")
-    val hogFactor = configurationDescriptor.globalConfig.getInt("system.hog-safety.hog-factor")
+    // if defined, use per-backend hog-factor, otherwise use system-level value
+    val hogFactor = configurationDescriptor.backendConfig.as[Option[Int]]("hog-factor") match {
+      case Some(backendHogFactorValue) => backendHogFactorValue
+      case None => configurationDescriptor.globalConfig.getInt("system.hog-safety.hog-factor")
+    }
```

Comment:
```suggestion
    val hogFactor = configurationDescriptor.backendConfig.getOrElse[Int]("hog-factor", configurationDescriptor.globalConfig.getInt("system.hog-safety.hog-factor"))
    }
```

Reply:
I think the version with pattern-matching is easier to read in this case

Reply:
Just as long as we don't use [`fold`](https://alvinalexander.com/scala/how-use-fold-scala-option-some-none-syntax/), I'm happy with either!